src/components/header.component/Header.jsx

```diff
@@ -0,0 +1,66 @@
+import "./header.css";
+export const Header = () => (
+  <header className="nav-header  nav-header--e-comm nav--e-comm-bg  w-p-full">
+    <nav className="nav nav--e-comm  nav--e-comm-bg flex-r flex-s flex-sp-eve ">
+      {/* <!-- Brand Name  --> */}
+      <a
+        className=" brand-name--e-comm text-md m-lr-md "
+        href="../../index.html"
+      >
+        PHOENIX<sub>SHOP</sub>
+      </a>
+      {/* <!-- Search Field  --> */}
```

Comment:
Nice commenting This increases the developers experience

Reply:
Done